be/src/olap/tablet_meta.cpp

```diff
@@ -101,6 +89,8 @@ TabletMeta::TabletMeta(int64_t table_id, int64_t partition_id,
     tablet_meta_pb.set_cumulative_layer_point(-1);
     tablet_meta_pb.set_tablet_state(PB_RUNNING);
     *(tablet_meta_pb.mutable_tablet_uid()) = tablet_uid.to_proto();
+    tablet_meta_pb.set_tablet_type(tabletType == TTabletType::TABLET_TYPE_MEMORY ?
```

Comment:
Normal upgrade procedure is upgrading BE process proceeding of upgrading FE.
When upgrading BE, tabletType may be not set.

Reply:
fixed